applications/GeoMechanicsApplication/custom_utilities/process_parameters.h

```diff
@@ -0,0 +1,37 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#pragma once
+
+#include "includes/kratos_parameters.h"
+
+#include <string>
+
+namespace Kratos
+{
+
+struct ProcessParameters
+{
+    Parameters parameters;
+    std::string name;
+
+    ProcessParameters(const Parameters& rParameters, const std::string& rName) :
```

Comment:
To make reading easier, I would put the name as the first parameter and the settings (parameters) second.